### PR TITLE
Using 4326 for administrative divisions

### PR DESCRIPTION
**Pull request by pgiraud**
With this pull request I tried to enhance performances of the processing so that it doesn't fail on demo server.

### CODE REVIEW

File: thinkhazard_processing/processing.py [hunk 1]
```diff
@@ -165,39 +159,36 @@ def create_outputs(hazardset, layers, readers):
...
         admindivs = admindivs \
-            .filter(
-                func.ST_Transform(AdministrativeDivision.geom, 4326)
-                .intersects(
+            .filter(AdministrativeDivision.geom.intersects(
                     func.ST_GeomFromText(bbox.wkt, 4326))) \
-            .filter(func.ST_Intersects(
-                func.ST_Transform(AdministrativeDivision.geom, 4326),
-                func.ST_GeomFromText(bbox.wkt, 4326)))
+            .filter(AdministrativeDivision.geom.intersects(
```
Comment (arnaud-morvan) [inline thinkhazard_processing/processing.py]: The original query was "intersects" (spatial index on bbox) and "ST_Intersects" (strict geometric predicate).
It seems you replace this by "intersects" and "intersects".

Comment (pgiraud) [inline thinkhazard_processing/processing.py]: Good catch. Indeed, my bad.

Comment (pgiraud) [inline thinkhazard_processing/processing.py]: I wonder if we really need both. Probably ST_intersects is smart enough to first compare bboxes then geometries.

Comment (arnaud-morvan) [inline thinkhazard_processing/processing.py]: Intersects use the spatial index to exclude most of the not relevant features (bbox do not intersect), ST_intersect filter the resulting ones that are not really relevant (geometries do not intersect). && AND ST_intersects is a common filter pattern with Postgis.

Comment (arnaud-morvan) [inline thinkhazard_processing/processing.py]: You are right: 
_This function call will automatically include a bounding box comparison that will make use of any indexes that are available on the geometries._

**Pull request merged**